tests/rt.conf

```diff
@@ -2,26 +2,31 @@
 # S2S tests                                                                                                                                                                    #
 ###################################################################################################################################################################################
 
-COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1                                                           | - wcoss_cray                            | fv3 |
+COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v16_coupled_rrtmgp                                | - wcoss_cray                            | fv3 |
 # Waves off
 RUN     | cpld_control_p7                                                                                                         | - wcoss_cray                            | fv3 |
+RUN     | cpld_control_p7_rrtmgp                                                                                                  | - wcoss_cray                            | fv3 |
```

Comment:
Did the cpld p7 rrtmp pass ORT test including the threading test?

Reply:
Yes. 2thread, debug, decomposition, and restart passed.